sdk/identity/platform-matrix.json

```diff
@@ -6,7 +6,7 @@
     {
       "Agent": {
         "msi_image": {
-          "OSVmImage": "env:LINUXNEXTVMIMAGE",
+          "OSVmImage": "env:LINUXVMIMAGE",
           "Pool": "azsdk-pool-mms-ubuntu-2204-identitymsi",
```

Comment:
@christothes do you know if this custom pool is still needed? What is special about it?

Reply:
It's used for Managed Identity live tests. @benbp set it up for me a while back so that it had a user-assigned managed identity associated with it.

Reply:
@christothes do you know if it is needed any longer given we use the container image? Or is this a separate set of tests? We would like to eliminate one-off agent pools if we can.

Reply:
created https://github.com/Azure/azure-sdk-for-net/issues/50546 to track.